lib/shelley/test/unit/Cardano/Wallet/Shelley/MinimumUTxOSpec.hs

```diff
@@ -0,0 +1,57 @@
+{-# LANGUAGE TypeApplications #-}
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Cardano.Wallet.Shelley.MinimumUTxOSpec
+    ( spec
+    ) where
+
+import Prelude
+
+import Cardano.Wallet.Primitive.Types.Coin
+    ( Coin (..) )
+import Cardano.Wallet.Primitive.Types.MinimumUTxO
+    ( MinimumUTxO )
+import Cardano.Wallet.Primitive.Types.MinimumUTxO.Gen
+    ( genMinimumUTxO, shrinkMinimumUTxO )
+import Cardano.Wallet.Primitive.Types.TokenMap
+    ( TokenMap )
+import Cardano.Wallet.Primitive.Types.TokenMap.Gen
+    ( genTokenMap, shrinkTokenMap )
+import Cardano.Wallet.Shelley.MinimumUTxO
+    ( computeMinimumCoinForUTxO )
+import Data.Function
+    ( (&) )
+import Test.Hspec
+    ( Spec, describe, it )
+import Test.QuickCheck
+    ( Arbitrary (..), Property, property )
+import Test.QuickCheck.Classes
+    ( eqLaws, showLaws )
+import Test.Utils.Laws
+    ( testLawsMany )
+
+spec :: Spec
+spec = do
+    describe "Class instances obey laws" $ do
+        testLawsMany @MinimumUTxO
+            [ eqLaws
+            , showLaws
+            ]
+
+    describe "computeMinimumCoinForUTxO" $
+        it "prop_computeMinimumCoinForUTxO" $
+            prop_computeMinimumCoinForUTxO
+                & property
+
+prop_computeMinimumCoinForUTxO :: MinimumUTxO -> TokenMap -> Property
+prop_computeMinimumCoinForUTxO minimumUTxO m = property $
+    computeMinimumCoinForUTxO minimumUTxO m >= Coin 0
```

Comment:
This is just a placeholder property. More properties will be added later. (See PR description.)